docs/development/plugins/building.md

```diff
@@ -39,6 +39,16 @@ npm run build
 This will create a file with the bundled plugin in
 `headlamp-myfancy/dist/main.js`.
 
+### building a folder of packages at once
+
+For convienience the `headlamp-plugin build` command can build a 
+package or folder of packages.
+
+```bash
+npx @kinvolk/headlamp-plugin build myplugins
+npx @kinvolk/headlamp-plugin build myplugins/headlamp-myfancy
```

Comment:
```suggestion
npx @kinvolk/headlamp-plugin build myplugins/headlamp-myfancy
npx @kinvolk/headlamp-plugin build myplugins
```

Make it respective to the comment above.

Should we include the "current" folder? Maybe that's already covered elsewhere.